src/components/Blocks/BlockNotFound.tsx

```diff
@@ -0,0 +1,25 @@
+import { Alert, AlertIcon, Code, Stack } from '@chakra-ui/react'
+import { ErrAPI } from '@vocdoni/sdk'
+import { Trans } from 'react-i18next'
+import { useRouteError } from 'react-router-dom'
+import RouteError from '~src/router/RouteError'
+
+export const BlockNotFound = () => {
+  const error = useRouteError() as Error
+
+  if (error instanceof ErrAPI && error.raw?.response?.status === 404) {
+    return (
+      <Stack spacing={4}>
+        <Alert status='warning'>
+          <AlertIcon />
+          <Trans i18nKey={'blocks.block_not_found'}>
+            The block you are trying to access has been pruned or it is not indexed
+          </Trans>
+        </Alert>
+        <Code>{error.message}</Code>
+      </Stack>
+    )
+  }
+
+  return <RouteError />
+}
```

Comment:
This file should be in src/router with other router elements like the `RouteError` used there, since it uses react-router-dom stuff and it's expected to be used as a router element.

We could consider creating a `src/router/errors` folder to have all the errors there.

Reply:
Agree 355747e